### PR TITLE
Isrumnon patch 2

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Get-CsTenant.md [hunk 1]
```diff
@@ -37,6 +37,64 @@ Get-CsTenant
 The command shown in Example 1 returns information about your tenant.
 Organizations will have only one tenant.
 
+**NOTE**
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTenant.md:40]:
```suggestion
**NOTE:**
```

File: skype/skype-ps/skype/Get-CsTenant.md [hunk 2]
```diff
@@ -37,6 +37,64 @@ Get-CsTenant
...
+
+Beginning TPM 2.6.2 onwards, the below updates are applicable for TeamsOnly customers
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTenant.md:42]:
```suggestion
Beginning TPM 2.6.2 onwards, the below updates are applicable for TeamsOnly customers.
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Please move this note from "Examples" section to "Description". Thanks.

**Review by dariomws — APPROVED** [review]
Thank you @isrumnon.

I think it's a good idea to replace TPM with "Microsoft Teams PowerShell version" or "Microsoft Teams PowerShell Module version" as @get-itips suggested.
With just TPM we can expect that someone opens an issue to ask about what the acronym means.

**Comment by isrumnon** [comment]
Done

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**